setup.py

```diff
@@ -1,102 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Could we just leave setup.py here for compatibility? Users might fallback to this file if anything is wrong and we can safely remove it after we receive enough feedback.

Reply:
Sure, we could note that users could use `pip install . --no-use-pep517` to force use of setup.py.

Reply:
Ironically, it seems that including both `setup.py` and `pyproject.toml` may cause headaches, as when `setuptools.build_meta` is called, it parses `pyproject.toml` first, but it runs additional code, *including the execution of setup.py (with a default stub if non-existent, which unfortunately does not apply in the current state of the PR)* to retrieve metadata and other information.

I'll file an issue with `setuptools`, and most likely their solution would just be a warning during build, but the easy fix is to replace `setup.py` with the `setup.cfg`, or to remove `setup.py` entirely.